### DISCUSSION TITLE
Got error using flak Flask==2.0.2

### DISCUSSION THREAD

**Discussion by bullet-dev-team** [pallets/flask#5277] [Q&A]
My application runs fine previously, but starting Sep 30, it failed 

I have Flask==2.0.2
gunicorn==20.0.4

**Comment by ThiefMaster** [comment]
Come on, do you expect us to have a crystal ball? "it failed" is about as useless as it gets. You need to include error messages, tracebacks, etc.

Anyway, a wild guess would be that you did not pin your dependencies and suddenly installed werkzeug 3.0 with an older flask version and/or still use deprecated code that has been removed in the latest major release...

**Reply by ghawes85** [reply]
@bullet-dev-team, can you try pinning Werkzeug==2.3.7 to your requirements under the flask install, and see if that fixes the issue? 3 days ago Werkzeug was bumped to 3.0.0, and other dependencies I use that also use Werkzeug caused the issue on my end.
